### PR TITLE
Diffs 🎉

### PR DESCRIPTION
**Pull request by davodesign84**
🎉

### CODE REVIEW

File: packages/kauri-components/components/Card/ArticleCard.tsx [hunk 1]
```diff
@@ -380,6 +380,7 @@ interface IProps {
   imageURL: string | null;
   cardHeight: number;
   cardWidth?: number;
+  destination?: string;
```
Comment (rej156) [inline packages/kauri-components/components/Card/ArticleCard.tsx]:
```suggestion
  destination?: 'review';
```

File: packages/kauri-components/components/Headers/ModalHeader.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const Container = styled.div`
 
 interface IProps {
   title: React.ReactElement<any>;
-  actions: React.ReactElement<any>;
+  actions?: React.ReactElement<any>;
```
Comment (rej156) [inline packages/kauri-components/components/Headers/ModalHeader.tsx]: What's this about 0_o

File: packages/kauri-web/components/containers/Article/View.js [hunk 3]
```diff
@@ -112,30 +110,7 @@ class Article extends React.Component<ArticleProps> {
...
-            addCommentAction={this.props.addCommentAction}
-            personalUsername={this.props.personalUsername}
-            publishArticle={this.publishArticle}
-            openModalAction={this.props.openModalAction}
-            closeModalAction={this.props.closeModalAction}
-          />
-        </>
-      );
-    } else if (R.path(["data", "getArticle", "status"])(this.props))
+    if (R.path(["data", "getArticle", "status"])(this.props))
```
Comment (rej156) [inline packages/kauri-web/components/containers/Article/View.js]: Shouldn't this be equal to something?

File: packages/kauri-components/components/Card/ArticleCard.tsx (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Card/ArticleCard.tsx]: Prefer enums/string literals ^^

File: packages/kauri-components/components/Headers/ModalHeader.tsx (hunk 2 above)
Comment (davodesign84) [reply packages/kauri-components/components/Headers/ModalHeader.tsx]: The current modal is broken in production I guess it expects a design similar to add to collection

File: packages/kauri-web/components/containers/Article/View.js (hunk 3 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/Article/View.js]: I just removed the old code and it's a different route :)

Comment (rej156) [reply packages/kauri-web/components/containers/Article/View.js]: Yeah fair, guess im still used to that withLoading HoC :+1:

**Pull request merged**